src/ndarray/ndarray.cc

```diff
@@ -31,10 +31,12 @@
 #include <mxnet/resource.h>
 #include <mxnet/imperative.h>
 #include <mshadow/tensor.h>
+#include <mkldnn.hpp>
```

Comment:
Should there be a macro before it's included?

Reply:
right